server/plugin.ts

```diff
@@ -60,6 +61,7 @@ export class OpendistroKibanaReportsPlugin
 
     // Register server side APIs
     registerReportRoutes(router);
+    registerReportDefinitionRoutes(router);
```

Comment:
try merge into one registerRoutes(router)
so we don't need to touch plugin.ts file when we introduce new router.

Reply:
Make sense, fixed